test-config/wdio.local.chrome.conf.js

```diff
@@ -99,7 +99,7 @@ exports.config = {
   connectionRetryCount: 3,
 
   services: [
-      ['selenium-standalone', { drivers: { chrome: true } }]
+      ['selenium-standalone', { drivers: { chrome: '106.0.5249.61' } }]
```

Comment:
Just curious, why a specific version, not the latest?

Reply:
In our case we apparently didn't have the latest version so we couldn't test locally, that's why we had to use that specific version. I will change it back and add a comment in case someone finds the same problem.